modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -53,6 +53,11 @@ The *Details* section of the *Pipeline Run Details* page displays a *Log Snippet
 * On the *Pipeline Run details* page, click the *Task Runs* tab to see the completed, running, and failed runs for the task.
 +
 The *Task Runs* tab provides information about the task run along with the links to its task and pod, and also the status and duration of the task run. Use the Options menu {kebab} to delete a task run.
++
+[NOTE]
+====
+The *TaskRuns* list page features a *Manage columns* button, which you can also use to add a *Duration* column.
```

Comment:
```suggestion
The *Task Runs* list page features a *Manage columns* button, which you can also use to add a *Duration* column.
```

Reply:
Hi @max-cx! The UI shows the *TaskRuns* instead of *Task Runs*. Is it okay to still change it?

Reply:
Of course, whatever is in the UI. The reason I made this suggestion was that the lines before and after in this file discuss a *Task Runs*, which is perhaps a different UI element. In any case, I wanted to bring this to your attention just to be safe.